### PR TITLE
Pass through loadbalancer annotations

### PR DESCRIPTION
**Pull request by Kidswiss**
Add ability to pass through annotations for the `primary-service` in PostgreSQL instances. Necessary for APPUiO managed instances.

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

File: component/tests/golden/vshn/appcat/appcat/21_composition_objectstorage_minio.yaml [hunk 1]
```diff
@@ -63,6 +64,7 @@ spec:
         apiVersion: v1
         data:
           providerConfig: minio-cluster
+          proxyEndpoint: host.docker.internal:9443
```
Comment (wejdross) [inline component/tests/golden/vshn/appcat/appcat/21_composition_objectstorage_minio.yaml:67]: why this proxyEndpoint?

**Comment by wejdross** [comment]
can You edit description?

File: component/tests/golden/vshn/appcat/appcat/21_composition_objectstorage_minio.yaml (hunk 1 above)
Comment (Kidswiss) [reply component/tests/golden/vshn/appcat/appcat/21_composition_objectstorage_minio.yaml:67]: ah whops, forgot to disable it before generating the goldens.

**Review by zugao — APPROVED** [review]
Missing docs, rest is fine.

**Pull request merged**